src/syntax/ppx_bisect.ml

```diff
@@ -0,0 +1,13 @@
+open Migrate_parsetree
+open Ppx_tools_404
+
+let () =
+  Driver.register ~name:"ppx_bisect" ~args:InstrumentArgs.switches
+    Versions.ocaml_404 (fun _config _cookies ->
+        (* let anon s = raise (invalid_arg ("nothing anonymous: " ^ s)) in *)
+        let anon _s = () in
```

Comment:
Looks like omp disallows anonymous arguments anyway btw:

https://github.com/let-def/ocaml-migrate-parsetree/blob/master/MANUAL.md#using-arguments-in-meta-ppxopt